FitGoal/FitGoal/Singup&Singin/CustomUIViews/AuthenticationLinkStack.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  LoginLinkStack.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 20/3/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+protocol AuthenticationTypeDelegate: AnyObject {
+    func userDidSelectAuthenticationType()
+}
```

Comment:
When you implement the `userDidSelectAuthenticationType` function, some times you display the login VC and others the signUp VC. It looks to me that a more natural name for this function would be `userDidSwitchAuthenticationType`.